data/Sduttt.json

```diff
@@ -0,0 +1,68 @@
+{
+    "name": "Subham Dutta",
+    "type": "personal",
+    "bio": "Web Developer | Technical Content Writer",
+    "tags": [
+        "React",
+        "Javascript",
+        "Typescript",
+        "NextJS"
+    ],
+    "socials": [
+        {
+            "icon": "FaTwitter",
+            "url": "https://twitter.com/Subhamd88404337"
+        },
+        {
+            "icon": "FaGithub",
+            "url": "https://github.com/Sduttt"
+        },
+        {
+            "icon": "FaLinkedin",
+            "url": "https://www.linkedin.com/in/subham-dutta-8670b8178/"
+        }
+    ],
+    "links": [
+        {
+            "name": "My Portfolio",
+            "url": "https://sduttt.netlify.app/",
+            "icon": "FaLink"
+        },
+        {
+            "name": "Hashnode: Read my Articles",
+            "url": "https://sdutta.hashnode.dev/",
+            "icon": "FaBlog",
+            "color": "#2962FF"
+        },
+        {
+            "name": "LinkedIn: Connect me",
+            "url": "https://www.linkedin.com/in/subham-dutta-8670b8178/",
+            "icon": "FaLinkedin",
+            "color": "#0E76A8"
+        },
+        {
+            "name": "Twitter: Follow me",
+            "url": "https://twitter.com/Subhamd88404337",
+            "icon": "FaTwitter",
+            "color": "#00ACEE"
+        }
+    ],
+    "milestones": [
+        {
+            "title": "Joined GitHub",
+            "date": "March 20122",
```

Comment:
The date value appears to be incorrect. Did you mean to use 2022 instead?

Reply:
Yes!! Sorry for the typing mistake.

Reply:
Hi @Sduttt, do you want to push a commit to update the date? 😃

Reply:
Hey @weilincheng , I have pushed the suggested changes.

Reply:
Awesome! I have approved the changes.